supertokens_python/recipe/session/cookie_and_header.py

```diff
@@ -283,14 +291,16 @@ def _set_token(
 
 
 def token_response_mutator(
+    request: BaseRequest,
     config: SessionConfig,
     token_type: TokenType,
     value: str,
     expires: int,
     transfer_method: TokenTransferMethod,
 ):
     def mutator(response: BaseResponse):
-        _set_token(
+        _ = _set_token(
```

Comment:
```suggestion
        set_token(
```

why assigning it to `_`? What diff does it make?

Reply:
linter flags this because now it is async function